### PR TITLE
Deprecate this repository

### PR DESCRIPTION
**Pull request by natemcmaster**
To simplify the build setup, we can put both the compile and packaging build steps into one repository.

We will still use separate build configurations on TeamCity to compile for osx/windows/linux etc, but keeping the build code in separate repositories isn't necessary.

Code moved here: https://github.com/aspnet/libuv-build/pull/26

### CODE REVIEW

File: content/License.txt [hunk 1]
```diff
@@ -1,58 +0,0 @@
-
```
Comment (moozzyk) [inline content/License.txt:1]: This file must not be deleted. It is being linked from: https://www.nuget.org/packages/Libuv/1.9.0 and https://www.nuget.org/packages/Libuv/1.9.1

**Review by moozzyk — CHANGES_REQUESTED** [review]

File: content/License.txt (hunk 1 above)
Comment (natemcmaster) [reply content/License.txt:1]: Would you be okay if we update its contents to "This file has moved, please see [link to new place]" ?

Comment (moozzyk) [reply content/License.txt:1]: I don't think I can make this decision. Let's talk to @Eilon tomorrow.

**Comment by Eilon** [comment]
Is this change necessary? What is motivating it?

**Comment by natemcmaster** [comment]
Motivated by converting sake to msbuild. We want to simplify how libuv builds, and part of that means we don't need two repos to build and package this library.

File: content/License.txt (hunk 1 above)
Comment (Eilon) [reply content/License.txt:1]: Yeah that's a no-no. Can we just keep 2 repos but still change away from sake to MSBuild? I get that it isn't pretty, but this stuff here almost never changes anyway.

**Comment by natemcmaster** [comment]
>  Can we just keep 2 repos

Still in favor of one repo. Two repos is overengineering. If we keep two, when I finish conversion to MSBuild, the only thing this repo would contain is a nuspec file and few dozen lines of powershell that call "nuget.exe pack". (Plus license.txt which is undeletable of course).

**Comment by moozzyk** [comment]
You could leave just a license in this repo and move everything else, no?

**Comment by natemcmaster** [comment]
Talked with @Eilon. We'll reverse the direction of the PRs. Move content here from libuv-build.

**Pull request closed without merging**